src/components/resource/_fields/custom-embargo/custom-embargo-fields.js

```diff
@@ -25,8 +46,7 @@ export default class CustomEmbargoFields extends Component {
           <Field
             name="customEmbargoValue"
             component={TextField}
-            type="number"
-            parse={value => (!value ? null : (Number.isNaN(Number(value)) ? '' : Number(value)))}
+            placeholder="Number"
```

Comment:
Could we add an `autoFocus` to this when it's a brand new field?

Reply:
Thanks @Robdel12 🙌 `ariaLabel` and `autoFocus` added.